controllers/items.js

```diff
@@ -0,0 +1,51 @@
+const { BadRequestError, NotFoundError } = require('../errors')
+const Item = require('../models/item')
+const {StatusCodes} = require('http-status-codes')
+
+const getAllItems = async (req, res) => {
+    const items = await Item.find().sort('-createdAt')
```

Comment:
getAllItems isn't filtering by userId.  This is a security issue because a user can get items createdBy a different user.